scripts/cut_release.sh

```diff
@@ -111,6 +111,7 @@ lint_and_test() {
 
     # Tests
     printf "${blue}Running tests...${end}"
+    unset OPENSSL_FORCE_FIPS_MODE
```

Comment:
if this is used, do we still need the `--openssl-legacy-provider` flag on `yarn test`? or does this only work when used together?